File: ifElse.cpp
#include <iostream>
#include <string>

int main() {
    std::string name1;
    std::cout << "Enter name 1: ";
    std::cin >> name1;

    if (name1 == "maayan") {
        std::cout << "Hello Maayan!" << std::endl;
    } else if (name1 == "Matan") {
        std::cout << "Hello Matan." << std::endl;
    } else if (name1 == "Yarden") {
        std::cout << "Hello Yardan." << std::endl;
    }

    return 0;
}

File: ifElse.java
import java.util.Scanner;

public class ifElse {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter name 1: ");
        String name1 = scanner.nextLine();

        if (name1.equals("maayan")) {
            System.out.println("Hello Maayan!");
        } else if (name1.equals("Matan")) {
            System.out.println("Hello Matan.");
        } else if (name1.equals("Yarden")) {
            System.out.println("Hello Yardan.");
        }
    }
}